src/App.css

```diff
@@ -39,19 +52,57 @@
   padding: 5px 0 20px 0;
 }
 
+.table {
+  padding: 30px;
+}
+
 tr {
   color: #5b5757;
 }
+td {
+  padding: 20px;
+  text-align: left;
+}
+
+tr:nth-child(even) {
+  background-color: #f2f2f2;
+}
 
 .results {
   padding-top: 15px;
 }
 
+.table-head {
+  background-color: #d2d2d2;
+  font-weight: bold;
+}
+
 .footer {
   padding-top: 20px;
+  margin-top: 50px;
   text-align: center;
 }
+.footer-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #222;
+  height: 50px;
+  padding: 20px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1em;
+  font-weight: bold;
+  bottom: 0;
+  margin-bottom: 0;
```

Comment:
Is the `bottom: 0;` line needed here? Also, what about `margin-bottom: 0;`? Try to remove each one individually and see what happens!

Reply:
I think you're right bottom=0 is not needed but `  margin-bottom: 0;` needed to remove the bottom margin under footer